cmd/common.go

```diff
@@ -237,6 +238,64 @@ func loadConfig(nodetype check.NodeType) string {
 	return filepath.Join(path, file)
 }
 
+func mapToCISVersion(kubeToCISMap map[string]string, kv string) string {
+	cisVersion, found := kubeToCISMap[kv]
+	for !found && (kv != defaultKubeVersion && !isEmpty(kv)) {
+		kv = decrementVersion(kv)
+		cisVersion, found = kubeToCISMap[kv]
+		glog.V(2).Info(fmt.Sprintf("mapToCISVersion for cisVersion: %q found: %t\n", cisVersion, found))
+	}
+
+	if !found {
+		glog.V(2).Info(fmt.Sprintf("mapToCISVersion unable to find a match for: %q", kv))
+		glog.V(2).Info(fmt.Sprintf("mapToCISVersion kubeToCISMap: %#v", kubeToCISMap))
+		return ""
+	}
+
+	return cisVersion
+}
+
+func loadVersionMapping(v *viper.Viper) (map[string]string, error) {
+	versionMappingConf := v.Sub(string(versionMapping))
+	if versionMappingConf == nil {
+		return nil, fmt.Errorf("config file is missing %s section", versionMapping)
+	}
+	kubeToCISMap := versionMappingConf.GetStringMapString(kubersionCISMapping)
+	if kubeToCISMap == nil || (len(kubeToCISMap) == 0) {
+		return nil, fmt.Errorf("config file is missing %s.%s section", versionMapping, kubersionCISMapping)
+	}
```

Comment:
Per comment above, if we just had `version_mapping` this function could be half the length